lara-typescript/src/section-authoring/components/authoring-page.tsx

```diff
@@ -153,21 +154,21 @@ export const AuthoringPage: React.FC<IPageProps> = ({
     selectedPosition: string,
     selectedOtherItemId: string
     ) => {
-    const itemToMoveIndex = items.findIndex(i => i.id === itemId);
-    const itemToMove = items[itemToMoveIndex];
+    const itemIndex = items.findIndex(i => i.id === itemId);
```

Comment:
At some point we should probably revisit this, and may be pull out the logic into a utility function.